### PR TITLE
first draft of data-syncing doc

### PR DESCRIPTION
**Pull request by gitaaron**
Related Issue : https://github.com/functionland/docs/issues/66

Please note the images referenced in the doc are under 'static/diagrams' which you  can access from 'files changed'.

### CODE REVIEW

**Review by masih — APPROVED** [review]
This is great thanks @gitaaron 👍 Is there an issue somewhere that captures the delivery of this RFC?

File: docs/RFCs/personal-data-syncing.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+![Newsroom FLow Chart Implementation](/diagrams/data-sync-newsroom.png)
+
+
+### Docker-Compose
+
+In order to access ipfs-cluster-ctl and ipfs from the command line, the correspondant and newsroom will be bundled in the same image as ipfs-cluster-ctl and IPFS.
+
+A separate service will be defined for:
+
+  * fula-api
```
Comment (farhoud) [inline docs/RFCs/personal-data-syncing.md:122]: By fula-api you mean functionland/box?

File: docs/RFCs/personal-data-syncing.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  * peers should not enter a recursive loop with each other by notifying authors of updates that originated from them
+
+  * each program should be reentrant in the sense that they can be aborted at any point during execution then restarted
+
+     * the shared state will be updated to reflect the latest changes to the file system
+
+
+### Correspondant
+
+The role of the correspondant is to ensure IPFS-Cluster is pinning the latest files created and updated on the author's file system.
```
Comment (farhoud) [inline docs/RFCs/personal-data-syncing.md:101]: ### For upload Scenario:
The @functionland/fula API Get a file and return a CID to user.
And in current implementation of [@functionland/box](https://github.com/functionland/fula/blob/main/apps/box/src/file/index.ts) we use js-ipfs to store stream and return the CID to the user. 
So can you describe the change needed to add to  @functionland/box to work.

**Comment by gitaaron** [comment]
> 

My bad @masih - here is the [issue](https://github.com/functionland/docs/issues/66) (also added above)

File: docs/RFCs/personal-data-syncing.md (hunk 1 above)
Comment (gitaaron) [reply docs/RFCs/personal-data-syncing.md:122]: correct 👍

**Pull request closed without merging**